templates/ui/dashboard.html

```diff
@@ -34,8 +34,8 @@ <h3>
 				</h3>
 				<img src="http://i.imgur.com/y8lfLmz.jpg" class="img-responsive img-thumbnail pull-right" id="img1">
 				<div>
-					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.		
-					<center>	
+					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.
+					<center>
```

Comment:
Please remove this change. Make sure you don't commit empty changes next time.

Reply:
I didn't do that formating and I don't know why it was formatted that way.